openwisp_monitoring/monitoring/migrations/0012_rename_graph_chart.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 3.0.3 on 2020-06-01 17:04
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('monitoring', '0011_alert_settings_contenttype_registration'),
+    ]
+
+    operations = [
+        migrations.AlterModelTable(name='graph', table='monitoring_chart',),
+        migrations.RenameModel(old_name='Graph', new_name='Chart',),
+        migrations.AlterModelTable(name='chart', table=None,),
```

Comment:
Why is this needed?

Reply:
There were two options either doing this or the second one was to declare  `db_table` for the model when renaming. I think declaring `db_table` in the `Meta` would be a better thing to do here (though I am unsure what is the general practice)

Reply:
I see you have now removed `migrations.AlterModelTable(name='chart', table=None,),` :+1: